### ISSUE TITLE
citation instructions

### ISSUE THREAD

**Issue by klkeys** [alanderos91/BioSimulator.jl#10]
The landing page for the Git repo should have citation instructions.

The published manuscript is [here](https://www.sciencedirect.com/science/article/pii/S0169260718301822?dgcid=coauthor).

Once the preprint is out, we can link to it too.

Also, we should have a BibTeX copy-paste box, something like
```
@article{Biosimulator,
	Author = {{Alfonso Landeros and Timothy Stutz and Kevin L. Keys and Alexander Alekseyenko and Janet S. Sinsheimer and Kenneth Lange and Mary E. Sehl}},
	Journal = {{Computer Methods and Programs in Biomedicine}},
	Pages = {23 -- 35},
	Title = {{BioSimulator.jl: Stochastic simulation in Julia}},
	Volume = {167},
	Year = {2018}
}
```

**Comment by klkeys** [comment]
In a similar vein, we should cite both DifferentialEquations.jl and JuMP (related to #9?)

**DifferentialEquations.jl**
Rackauckas, C. and Nie, Q., 2017. DifferentialEquations.jl – A Performant and Feature-Rich Ecosystem for Solving Differential Equations in Julia. Journal of Open Research Software, 5(1), p.15. DOI: http://doi.org/10.5334/jors.151

**JuMP** (from git repo [here](https://github.com/JuliaOpt/JuMP.jl))
```
@article{DunningHuchetteLubin2017,
author = {Iain Dunning and Joey Huchette and Miles Lubin},
title = {JuMP: A Modeling Language for Mathematical Optimization},
journal = {SIAM Review},
volume = {59},
number = {2},
pages = {295-320},
year = {2017},
doi = {10.1137/15M1020575},
}
```

**Comment by klkeys** [comment]
Preprint is live ([link](https://arxiv.org/abs/1811.12499)).

**Comment by klkeys** [comment]
Closed #12.
